core/store/migrate/migrations/0092_terra_txm.sql

```diff
@@ -1,5 +1,22 @@
 -- +goose Up
 -- +goose StatementBegin
+ALTER TABLE heads RENAME TO evm_heads;
+ALTER TABLE nodes RENAME TO evm_nodes;
+-- CREATE TABLE terra_chains (
```

Comment:
I think we do want this table, is enabling it planned for a follow up PR?

Reply:
Yeah was just getting ahead of myself.